src/cli.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+const Configstore = require('configstore');
+const program = require('commander');
+
+const pkg = require('../package.json');
+const inquirer = require('./inquirer');
+const configStore = new Configstore(pkg.name, {
+  url: 'https://api.github.com',
+});
+
+program
+  .version(pkg.version, '-v, --version')
+  .option('--init', `Initialize ${pkg.name}`)
+  .option('-i, --issues', 'Include issues')
+  .option('-p, --pull-requests', 'Include pull requests')
+  .option('-c, --commits', 'Include commits')
+  .option('-t, --timeframe', 'The timeframe in hours')
+  .option('-g, --github-url [url]', 'The GitHub URL', 'https://api.github.com')
```

Comment:
what would be the value of `program.githubUrl` if I ran this file with the `-g` option as opposed to `-github-url api.ibm.com` ?